### ISSUE TITLE
TIDA popup rendering mangles content with styles

### ISSUE THREAD

**Issue by micheal-cooper** [ywc/ywc#3]
It seems that YWC goes nuts if you have a script block of styles for layout.

I have attached a screenshot of what the Event page looks like in Drupal, and I have pasted in below it a text file of the code in the content column of that Event node.

It appears that YWC is choking on the custom CSS used. The expected width of the window is of course a problem, but YWC is also showing the embedded CSS and repeating the contents of blocks. I don't understand why.

I will do emergency surgery make this visible on TIDA, but the ideal is that each event gets put in once and appears on our public Drupal website, our internal Drupal calendar, and on TIDA. For things like this concert,  the Event node is the online poster, as it were, so we are asked to make it look good. I am sure that other YWC installations will have similar situations.

If it cannot or should not be fixed, we make it a part of the rules for using YWC. if it can be fixed, then we should.

As always, thank you.
![piano concert page screenshot](https://f.cloud.github.com/assets/3095809/428526/6d10edc0-ae0f-11e2-84b8-b2cec3b7e3f5.png)

<pre>
<style type="text/css">
.center-block {
    margin:0px 0px 20px 0px;
    text-align:center;
    clear:both;
}
.red-notice-big {
    color:#c80019;
    font-size:28px;
}
.red-notice-med {
    color:#c80019;
    font-size:20px;
}
.blk-notice-big {
    color:#000000;
    font-size:20px;
}
.blk-notice-med {
    color:#000000;
    font-size:12px;
}</style>
<h2>広瀬悦子　ピアノソロコンサート</h2>
<p>OISTコンサートシリーズ No. 5 / OIST Concert Series No. 5</p>
<div class="center-block"><span class="red-notice-big">5/10 (Fri 金)</span> &nbsp;&nbsp;&nbsp;<span class="blk-notice-med">開場 Open</span> <span class="blk-notice-big">19:00</span> &nbsp;&nbsp;&nbsp;<span class="blk-notice-med">開演 Start</span> <span class="blk-notice-big">19:30</span> &nbsp;&nbsp;&nbsp;<span class="red-notice-med">Admission Free 入場無料</span><br>
    <span class="blk-notice-big">OIST Auditorium &nbsp; &nbsp;講堂</span></div>
<div style="width:370px; margin:0px 0px 0px 0px; float:left;">
    <h4>演目 Program</h4>
    <p>バッハ＝リスト編曲：前奏曲とフーガ　イ短調　BWV543<br>
        Bach=Liszt: Prelude and Fugue in A minor BWV 543</p>
    <p>ショパン：バラード第2番　op.38<br>
        Chopin: Ballade no.2 op.38</p>
    <p>ショパン：夜想曲　op.9-2<br>
        Chopin: Nocturne op.9-2</p>
    <p>ショパン：バラード第1番　op.23<br>
        Chopin: Ballade no.1 op.23</p>
    <p>フォーレ：夜想曲第6番　op.63<br>
        Fauré : Nocturne no.6 op.63</p>
    <p>アルカン：練習曲「鉄道」　op.27<br>
        Alkan : Le chemin de fer op.27</p>
    <p>ラヴェル：海原の小舟<br>
        Ravel : Une barque sur l'océan</p>
    <p>デュティユー：コーラルと変奏曲<br>
        Dutilleux : Choral et Variations</p>
</div>
<div style="width:320px; margin:50px 0px 0px 20px; float:right;"><img alt="" src="/sites/default/files/img/cpr/events/20130418-piano-hirose-Img2711-s.jpg" style="width: 320px; height: 481px;"></div>
<div style="clear:both;">&nbsp;</div>
<h4>広瀬悦子 Etsuko Hirose</h4>
<p style="width:350px;margin:0px 20px 20px 0px; float:left;">1992年モスクワ青少年ショパン国際ピアノコンクール優勝、1999年マルタ・アルゲリッチ国際コンクールで優勝。1999年パリ国立高等音楽院卒業以来、日本、フランス、アルゼンチンポーランド、イタリアなどで活躍。</p>
<p style="width:350px;margin:0px 0px 20px 0px; float:left;">Hirose studied at the Ecole Normale de Musique in Paris, France. In her career, she has won numerous prestigious international contests, such as the Frederic Chopin Competition (1992) and the Martha Argerich Competition (1999). She has given performances around the world, including in Japan, France, Argentina, Poland and Italy.</p>
<div style="clear:both;">&nbsp;</div>
<p class="center-block">（全席自由・先着順） チケットは5月1日（水）10時より予約開始<br>
    Tickets available from 10:00 AM Wednesday, May 1st on a first-come, first-served basis<br>
    <a href="http://www.oist.jp/reserve"><span class="red-notice-med">予約する（5月1日10時より）<br>
    Reserve Tickets (After 10am May 1st)</span></a></p>

</pre>

**Comment by topherwhite** [comment]
@micheal-cooper 
Can you confirm if this is still an issue? Thanks!

**Comment by micheal-cooper** [comment]
Yes, it is still an issue, but I just added it to the GitHub issue queue.
